ParabolInc/parabol
# feat: Updated new meeting settings area

Fixes #5603

**Next PR: #7001**

<img width="919" alt="image" src="https://user-images.githubusercontent.com/466991/182443290-6f99b637-9fbd-4622-aa3e-fcea920ad8ad.png">

<img width="361" alt="image" src="https://user-images.githubusercontent.com/466991/182443361-8553af5e-bcfb-4bfb-b651-24f256a3256e.png">

How to test:

- Open new meeting dialog
- Make sure all new controls works for all meeting types bot on mobile and desktop
